File: python-demo/app/service/user_service.py
from app.model.user import User

class UserService:

    def __init__(self):
        self.users = [
            User(id=1, name="Tom", age=18)
        ]

    def list_users(self):
        return self.users

    def add_user(self, user: User):
        self.users.append(user)
        return user

File: node-demo/src/service/UserService.ts
import { User } from "../model/User";

export class UserService {

    private users: User[] = [
        {
            id: 1,
            name: "Tom",
            age: 18
        }
    ];

    public getAll(): User[] {
        return this.users;
    }

    public save(user: User): User {

        this.users.push(user);

        return user;
    }
}